pkg/synthesis/synthesis_test.go

```diff
@@ -193,8 +193,13 @@ func addDebugFiles(t *testing.T, rc *collector.ResourcesContainerModel, abstract
 		require.Nil(t, err)
 	}
 	// write the abstract model rules into a file
-	actualOutput := strAllowOnlyPolicy(abstractModel.policy[0])
-	err = common.WriteToFile(path.Join(debugDir, "abstract_model.txt"), actualOutput)
+	abstractStr := strAllowOnlyPolicy(abstractModel.policy[0])
+	err = common.WriteToFile(path.Join(debugDir, "abstract_model.txt"), abstractStr)
+	require.Nil(t, err)
+	// write the config summery into a file
+	configStr, err := model.NSXConfigStrFromResourcesContainer(rc, common.OutputParameters{})
+	require.Nil(t, err)
+	err = common.WriteToFile(path.Join(debugDir, "config.txt"), configStr)
```

Comment:
the configStr is part of the log output (already printed to stderr on the previous call to `NSXConnectivityFromResourcesContainer`).

maybe (consider for separate issue) better extend `OutputParameters` to allow configuring log file,  so all log output is also stored in a specified file?